src/__experimental__/components/decimal/decimal.stories.mdx

```diff
@@ -83,18 +92,27 @@ import Decimal from "carbon-react/lib/__experimental__/components/decimal";
 
 <Preview>
   <Story name="labelAlign">
-    {() =>
+    {() => {
+      const [state, setState] = useState({
+        right: "0.01",
+        left: "0.01",
+        large: "0.01",
```

Comment:
Don't think you need `large`. The story also doesn't render when I ran the branch

![image](https://user-images.githubusercontent.com/44157880/119162424-41dcb500-ba52-11eb-85cd-46c42a846c93.png)